apps/web/src/components/Builders/Appchains/InfoCards.tsx

```diff
@@ -0,0 +1,114 @@
+import { Card, CardProps } from 'apps/web/src/components/Builders/Shared/Card';
+import lightningBolt from 'apps/web/src/components/Builders/Appchains/lighteningBolt.svg';
+import taxesReceipt from 'apps/web/src/components/Builders/Appchains/taxesReceipt.svg';
+import perpetualSwap from 'apps/web/src/components/Builders/Appchains/perpetualSwap.svg';
+import faces from 'apps/web/src/components/Builders/Appchains/faces.svg';
+import complianceProduct from 'apps/web/src/components/Builders/Appchains/complianceProduct.svg';
+import globe from 'apps/web/src/components/Builders/Appchains/globe.svg';
+import Image, { StaticImageData } from 'next/image';
+
+const INFO_CARDS: CardProps[] = [
```

Comment:
won't block on it, but i'm a pretty strong advocate against this pattern. since you already know the contents, there's no reason to create an array and then map over it. just put all the info directly in the cards.